check/pytest

```diff
@@ -19,7 +19,7 @@ cd "$(git rev-parse --show-toplevel)"
 
 PYTEST_ARGS=()
 ACTUALLY_QUIET=""
-for arg in "$@"; do
+for arg in $@; do
```

Comment:
I guess this is because the quoted version doesn't allow you to pass both flags?  This seems to hav been changed in correctly in #4629?

Reply:
Whoops, this shouldn't have been in this PR. I've had to change this locally because for some reason it doesn't work on my machine, but we've confirmed that issue is localized to my setup.

Reply:
Reverted.